.ci/aarch64/install_rom_aarch64.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+source /etc/os-release || source /usr/lib/os-release
+
+EDK2_REPO="https://github.com/tianocore/edk2.git"
+EDK2_PLAT_REPO="https://github.com/tianocore/edk2-platforms.git"
+ACPICA="https://github.com/acpica/acpica.git"
+
+TOOLCHAIN_SOURCE_URL="https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-a/9.2-2019.12/binrel/gcc-arm-9.2-2019.12-x86_64-aarch64-none-elf.tar.xz"
```

Comment:
This is one scary looking URL :smile: Is this the "most canonical" URL there is for this download?

Also:

- It might be worth adding a comment explaining the confusing filename is due to this being a cross compler.
- Can you use some variables to make maintaining this easier (for example the "double version number" in that URL [`9.2-2019.12`]).

Reply:
I find it is possible to build uefi on arm64 machine, so I don't need this odd URL 😄